Marlin/src/MarlinCore.cpp

```diff
@@ -1013,11 +1013,31 @@ void setup() {
   SETUP_RUN(HAL_init());
 
   // Init and disable SPI thermocouples
-  #if HEATER_0_USES_MAX6675
-    OUT_WRITE(MAX6675_SS_PIN, HIGH);  // Disable
+  #if HEATER_0_USES_MAX6675 && PIN_EXISTS(MAX6675_SS)
+    OUT_WRITE(MAX6675_SS_PIN, HIGH);
   #endif
-  #if HEATER_1_USES_MAX6675
-    OUT_WRITE(MAX6675_SS2_PIN, HIGH); // Disable
+  #if HEATER_1_USES_MAX6675 && PIN_EXISTS(MAX6675_SS2)
+    OUT_WRITE(MAX6675_SS2_PIN, HIGH);
+  #endif
+  #if HEATER_0_USES_MAX6675 && PIN_EXISTS(MAX6675_CS)
```

Comment:
Why support CS and SS pin names, when both have the same purpose? I suggest choosing just `CS` and moving all code for that.

Reply:
@rhapsodyv A lot of Marlin users that already have these boards setup will not be happy with the inability for the code to not work because we change the name.  I just think that using the MAX6675 pin names for boards that are not named MAX6675 is confusing.  I will let @thinkyhead  make the decision on this.   But for the time being let's get other things straighten out before I start removing the OLD names

Reply:
We can just add a compatibility layer on the first marlin conditionals post: ```
Ifdef oldname
 define Newname oldname
Endif
```
So, the entire code just need to look at one set of names...
It will make everything clear and easy to understand and maintain

Reply:
So I do need to go into conditional_post and do that?

Reply:
Well it looks like @thinkyhead has been busy with updating this stuff.  He has used the ANY_PIN macro

Reply:
@rhapsodyv Since the MAX31865 uses MAX6675_SS_PIN and the MAX31855 uses the MAX6675_SS_PIN and the MAX6675 of course uses the MAX6675_SS_PIN when you do the 
```
Ifdef MAX6675_SS_PIN
 define Newname MAX6675_SS_PIN
Endif
```
Which Newname are you going to use?  Either way you still have to do some more #if and comparisons.  @thinkyhead has used the ANY_PIN macro to make the code more readable

Reply:
you could us CS on marlin:
```cpp
#if PIN_EXISTS(MAX6675_CS)
   #define MAX6675_SS MAX6675_CS
#endif

#if PIN_EXISTS(MAX6675_CS2)
   #define MAX6675_SS2 MAX6675_CS2
#endif
```
And for MAX31855 too

Reply:
Yes, we could. Let me look at what @thinkyhead has done on this already and see what state the current code is in.

Reply:
Fixed this only MAX????_SS_PIN is now put into MAX????_CS_PIN in conditionals_post.h file so when temperature.cpp or MarlinCore.cpp code is run they both will use the MAX????_CS_PIN variables.